src/clt_least_squares/InputForm.jsx

```diff
@@ -65,101 +65,94 @@ export const InputForm = ({seed, handleSeed, handleGeneratePop, beta,
+                { seed &&
+                <div className={'form-group'}>
+                    <div className={'form-row'}>
+                        <label htmlFor={'slope'}>
+                            Beta:
+                        </label>
+                        <NumericField
+                            id={'beta'}
+                            className={'form-control form-control-sm'}
+                            min={0}
+                            max={1}
+                            value={beta}
+                            step={0.01}
+                            onChange={hndlBeta} />
+                    </div>
+                    <div className={'form-row'}>
+                        <div style={{ height: '50px', width: '100%'}}>
+                            <Rheostat
                                 min={0}
-                                max={1}
-                                value={beta}
-                                step={0.01}
-                                onChange={hndlBeta} />
-                        </div>
-                        <div className={'form-row'}>
-                            <div style={{ height: '50px', width: '100%'}}>
-                                <Rheostat
-                                    min={0}
-                                    max={99}
-                                    values={[(beta * 100)]}
-                                    onValuesUpdated={(sliderState) => {
-                                        handleBeta(
-                                            (sliderState.values[0] * 0.01));
-                                    }} />
-                            </div>
+                                max={99}
+                                values={[(beta * 100)]}
+                                onValuesUpdated={(sliderState) => {
+                                    handleBeta(
+                                        (sliderState.values[0] * 0.01));
+                                }} />
                         </div>
-                        <div className={'form-row'}>
-                            <label htmlFor={'alpha'}>
-                                Alpha:
-                            </label>
-                            <NumericField
-                                id={'alpha'}
-                                className={'form-control form-control-sm'}
+                    </div>
+                    <div className={'form-row'}>
+                        <label htmlFor={'alpha'}>
+                            Alpha:
+                        </label>
+                        <NumericField
+                            id={'alpha'}
+                            className={'form-control form-control-sm'}
+                            min={0}
+                            max={1}
+                            step={0.01}
+                            value={alpha}
+                            onChange={hndlAlpha} />
```

Comment:
Seeing exceptions when changing this NumericField (and others) via the input spinner. `hndlAlpha` seems to expect an event as the first parameter, but a numeric value is coming through instead. Example: InputForm.jsx, Ln. 46, `e.preventDefault()` blows with `Uncaught TypeError: e.preventDefault is not a function`. Changing the value through the slider works correctly.

Reply:
Fix it, thanks!